### PR TITLE
docs(repo): update readme installation and usage

### PR DESCRIPTION
**Pull request by heitortsergent**
@philsturgeon related to this [other PR](https://github.com/stoplightio/spectral/pull/2187#discussion_r938855924), I took a pass at making it clearer in the README that the user needs to create a local ruleset to run `lint` without passing the `--ruleset` parameter.

**Checklist**

- [ ] Tests added / updated
- [x] Docs added / updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,47 +9,52 @@
...
-Find more [installation methods](https://meta.stoplight.io/docs/spectral/ZG9jOjYyMDc0Mw-installation) in our documentation.
+You can find more installation methods [here](https://meta.stoplight.io/docs/spectral/ZG9jOjYyMDc0Mw-installation).
+
+## 💻 Usage
 
-**Create a local ruleset**
+### 1. Create a local ruleset
 
-Spectral, being a generic YAML/JSON linter, needs a ruleset to lint files. There are two ways to get a ruleset:
+Spectral, being a generic YAML/JSON linter, **needs a ruleset** to lint files. A ruleset is a JSON or YAML file (often the file will be called `.spectral.yaml`) that contains a collection of rules, which can be used to lint other JSON or YAML files such as an API description.
```
Comment (philsturgeon) [inline README.md]: A ruleset could also be JavaScript or TypeScript. Our docs for this need work but you can see how I've done it over here. https://apisyouwonthate.com/blog/distribute-spectral-style-guides-with-npm#publish-your-spectral-style-guide-to-npm

**Review by philsturgeon — DISMISSED** [review]
Only one comment but I think we need to have a chat about how we document the JS/TS rulesets properly anyway, so we could take care of that in another issue. Feel free to shove some Tuesday/Thursday time on my calendar to talk about how we document this.

File: README.md (hunk 1 above)
Comment (heitortsergent) [reply README.md]: Mhm, what do you think is the best way to word this? Maybe we just add JavaScript/TypeScript:

"Spectral, being a generic YAML/JSON linter, **needs a ruleset** to lint files. A ruleset is a JSON, YAML, or JavaScript/TypeScript file (often the file will be called `.spectral.yaml` for a YAML ruleset) that contains a collection of rules, which can be used to lint other JSON or YAML files such as an API description."

Comment (philsturgeon) [reply README.md]: @heitortsergent that works.

**Comment by heitortsergent** [comment]
@philsturgeon alright, made the change and set up some time for us to chat about JS/TS for Thursday. :)

**Comment by heitortsergent** [comment]
@mnaumanali94 / @P0lip would you mind taking a look at this please? It's a small update to make the docs a little more accurate.

**Review by P0lip — DISMISSED** [review]

**Review by mnaumanali94 — DISMISSED** [review]

**Review by pamgoodrich — DISMISSED** [review]

**Review by pamgoodrich — APPROVED** [review]

**Pull request merged**